Resources/Prototypes/Catalog/Bounties/bounties.yml

```diff
@@ -707,3 +707,381 @@
       tags:
       - MicrowaveMachineBoard
 
+- type: pirateBounty
+  id: BountyMail
+  reward: 2
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-mail
+    amount: 10
+    whitelist:
+      components:
+      - Mail
+    blacklist:
+      tag:
+      - Trash
+
+- type: pirateBounty
+  id: BountyPDA
+  reward: 2
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-pda
+    amount: 8
+    whitelist:
+      id:
+      - BasePDA
+
+- type: pirateBounty
+  id: BountyExtinguisher
+  reward: 2
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-extinguisher
+    amount: 5
+    whitelist:
+      id:
+      - FireExtinguisher
+
+- type: pirateBounty
+  id: BountyCaptainGloves
+  reward: 2
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-captainGloves
+    amount: 5
+    whitelist:
+      id:
+      - ClothingHandsGlovesCaptain
+
+- type: pirateBounty
+  id: BountyGyro
+  reward: 2
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-gyro
+    amount: 4
+    whitelist:
+      id:
+      - ShuttleGyroscope
+
+- type: pirateBounty
+  id: BountyDefib
+  reward: 3
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-defib
+    amount: 5
+    whitelist:
+      id:
+      - BaseDefibrillator
+
+- type: pirateBounty
+  id: BountyResearchDisk
+  reward: 5
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-researchDisk
+    amount: 7
+    whitelist:
+      id:
+      - ResearchDisk
+
+- type: pirateBounty
+  id: BountyAlcohol
+  reward: 2
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-alcohol
+    amount: 4
+    whitelist:
+      id:
+      - BoozeDispenser
+
+- type: pirateBounty
+  id: BountyThruster
+  reward: 4
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-thruster
+    amount: 4
+    whitelist:
+      id:
+      - BaseThruster
+
+- type: pirateBounty
+  id: BountyGravGen
+  reward: 2
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-gravGen
+    amount: 4
```

Comment:
should be 2 gravgens for 5DB, needing to steal 4 crews for only 2 DB is too few.